wallet/src/main/java/com/android/identity_credential/wallet/navigation/WalletDestination.kt

```diff
@@ -167,9 +171,6 @@ sealed class WalletDestination(val routeEnum: Route) : DestinationArguments() {
 }
 
 
-abstract class DestinationArguments {
```

Comment:
This change is here because when I approached the Credential Offer feature using a composable Screen I had to make changes to `WalletDestination` and this change makes the WalletDestination data structure more correct because the `abstract class DestinationArguments` was designed for Destinations of `WalletDestination` who have defined their own `enum Argument` and use  the `DestinationArguments` contract to provide a means for obtaining a list of all Argument enums it has defined. 

The chances of downstream customers inheriting from `DestinationArguments` outside of `WalletDestination` class should always be zero 0%.